receiver/prometheusreceiver/testdata/openmetrics/bad_missing_or_extra_commas_2/metrics

```diff
@@ -1,2 +0,0 @@
-a{a="1",b="2",} 1
```

Comment:
Could we keep this test but verify this case is working?

Reply:
added back as a success vase.